# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gpart/instpath.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./inst.defs.org	Wed Dec 13 23:45:49 2000
+++ ./inst.defs	Tue Apr 29 12:32:54 2003
@@ -1,7 +1,7 @@
 #
 # installation directories for gpart
 #
-prefix=/usr/local
+prefix=/usr
 bindir=$(prefix)/bin
 libdir=$(prefix)/lib
 mandir=$(prefix)/man
